src/utils/daos_hdlr.c

```diff
@@ -1645,6 +1656,84 @@ fs_copy_file(struct cmd_args_s *ap,
 	return rc;
 }
 
+static int
+fs_copy_symlink(struct cmd_args_s *ap,
+		struct file_dfs *src_file_dfs,
+		struct file_dfs *dst_file_dfs,
+		struct stat *src_stat,
+		const char *src_path,
+		const char *dst_path)
+{
+	int		rc = 0;
+	daos_size_t	len = 0;
+	char		*symlink_value = NULL;
+
+	D_ALLOC(symlink_value, DFS_MAX_PATH);
+	assert(symlink_value != NULL);
```

Comment:
You can't use assert() for something that could happen.

Reply:
Thanks! Return -DER_NOMEM now.